### ISSUE TITLE
DRAFT: Integrate a user survey

### ISSUE THREAD

**Issue by mmcfarland** [azavea/fb-gender-survey-dashboard#66]
# Draft

This is a draft issue, and it will need to clarify specs and formalize what
counts as "done" before work begins on it.

Check with the issue's creator before starting work on this issue.

## Overview

Integrate a user survey. Details TBD.

**Comment by mmcfarland** [comment]
Closing in favor of #69
